src/DGtal/topology/Object.h

```diff
@@ -81,7 +97,7 @@ namespace DGtal
    * adjacency relations are displayed.
    *
    * \b Model of CUndirectedSimpleLocalGraph and CUndirectedSimpleGraph.
-   * 
+   *
```

Comment:
I suggest you also reference the user module "interface with boost::graph library", since ObjectBoostGraphInterface.h is hard to read/understand for the newbie.